Sc question response interface

symbyte opened the pull request:
This is the preliminary design for the question response interface for students. It was designed with Material Design standards in mind, but shold also be backwards compatible to API 18.

Commit:
Add card views to gradle build and Implement simple card view

This card view will contain the question response interface for the student to answer a question broadcast by a teacher.

Commit:
extract question response card into its own layout

This commit adds some formatting to the question response card layout and extracts it into its own layout file for reusability

Commit:
finish preliminary front end layout for question responses

The question layout is currently hard wired for 4 responses as that will be default. The layout can be easily changed programmatically when/if we incorporate a question model class. Best effort was made here to follow google's material design guidelines.

Commit:
finish implementing fragment handling of UI elements

colegarien commented:
Why was build target lowered from api 23 to api 21?

Maybe I am reading the first commit wrong.

symbyte commented:
I didn't think about the fact that I was using a slightly older version of the support library in the dependacies and gradle complained about the build target so I changed it to get it to compile. I'll push a commit change to the newest version of the support library and change the build target back to 23. Sorry about that.

Commit:
change build target back to API 23

Switched to using a newer version of the support library and set the build target back to API 23

colegarien commented:
Looks good, will test then merge
